hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/OzoneRpcClientTests.java

```diff
@@ -231,17 +231,13 @@ abstract class OzoneRpcClientTests extends OzoneTestBase {
       remoteGroupName, ACCESS, READ);
   private static MessageDigest eTagProvider;
   private static Set<OzoneClient> ozoneClients = new HashSet<>();
-  private static PrintStream old;
-  private static ByteArrayOutputStream baos;
+  private static GenericTestUtils.PrintStreamCapturer out;
```

Comment:
Checkstyle complains about some local variables (in existing tests) hiding this new field.  So I guess the simplest fix is to rename the new field to `output`.